torchvision/datasets/video_utils.py

```diff
@@ -320,22 +326,22 @@ def get_clip(self, idx):
             end_pts = clip_pts[-1].item()
             video, audio, info = read_video(video_path, start_pts, end_pts)
         else:
-            info = _probe_video_from_file(video_path)
-            video_fps = info.video_fps
+            _info = _probe_video_from_file(video_path)
```

Comment:
Since we now have

https://github.com/pytorch/vision/blob/59baae99b3e1bd2e43b2c3aebf44505c5cdf1c90/mypy.ini#L6

this renaming is probably not needed anymore. Could you check?

Reply:
I did and unfortunately it's still an issue

Reply:
It seems even with `allow_redefinition`, `mypy` is [stricter than I thought](https://mypy.readthedocs.io/en/stable/config_file.html?highlight=allow_redefinition#confval-allow_redefinition):

> Allows variables to be redefined with an arbitrary type, **as long as the redefinition is in the same block and nesting level as the original definition.** [emphasis mine]

Since that is not the case here, we need to keep your fix.